tetrateci/test_1.13.sh

```diff
@@ -31,13 +31,11 @@ if [[ "${CLUSTER}" == "gke" ]]; then
 
   COMMON_TEST_FLAGS+=( "-istio.test.kube.helm.iopFile=${SCRIPTDIR}/iop-gke-integration.yml" )
 
-  echo "Applying GKE specific patches...."
-  git apply "${SCRIPTDIR}/patches/gke/chiron-gke.patch"
```

Comment:
Is this just not needed for 1.13?

Reply:
i see the test-cases were passing, so did not patched.

2022-04-08T14:16:03.005835Z	info	tf	=== DONE (passed):  Test: 'security_chiron[TestDNSCertificate/rotateDNSCertificatesWhenCertExpired] (5.072950013s)' ===
2022-04-08T14:16:03.005884Z	info	tf	=== DONE (passed):  Test: 'security_chiron[TestDNSCertificate] (15.3082972s)' ===
--- PASS: TestDNSCertificate (15.31s)
    --- PASS: TestDNSCertificate/generateDNSCertificates (0.06s)
    --- PASS: TestDNSCertificate/regenerateDNSCertificates (5.11s)
    --- PASS: TestDNSCertificate/rotateDNSCertificatesWhenCAUpdated (5.07s)
    --- PASS: TestDNSCertificate/rotateDNSCertificatesWhenCertExpired (5.07s)
PASS
2022-04-08T14:16:03.005913Z	info	tf	=== DONE: Test Run: 'security_chiron' ===
2022-04-08T14:16:03.005923Z	info	tf	=== Suite "security_chiron" run time: 45.618731486s ===
2022-04-08T14:16:03.216297Z	info	tf	=== BEGIN: Cleanup Istio [Suite=security_chiron] ===
2022-04-08T14:16:03.216331Z	info	tf	clean up cluster cluster-0
2022-04-08T14:16:34.841214Z	info	tf	=== SUCCEEDED: Cleanup Istio in 31.624826643s [Suite=security_chiron] ===
ok  	istio.io/istio/tests/integration/security/chiron	77.569s